### DISCUSSION TITLE
Connect to Bluetooth timers (such as GAN Smart Timer)

### DISCUSSION THREAD

**Discussion by raaahbin** [CubeLabsNZ/CubeTime#180] [Ideas]
I believe this should be possible as both cstimer and cubedesk web apps are able to do this when run in chrome… presumably the method to achieve this can even be found in that source code? Friends have recommended this app and it looks perfect apart from currently being unable to connect to an external timer.

**Comment by xbjfk** [comment]
Hey!
Thank you for your request - unfortunately both of us are really busy, so we may not get this done for a while, but it is absolutely on our to-do list!
